modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+container::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
+
+Image Registry Operator::
+The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
+
+image repository::
+An image repository is a collection of related container images and tags identifying images.
+
+mirror registry::
+A registry that holds the mirror of {product-title} images.
+
+namespace::
+A namespace isolates specific system resources that are visible to all processes. Inside a namespace, only processes that are members of that namespace can see those resources.
+
+{product-title} registry::
+{product-title} registry is a registry provided by {product-title} to manage images.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod is comprised of one or more containers to run in a worker node.
+
+private registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which allows the developers to push and pull their private container images.
+
+public registry::
+{product-title} can use any server implementing the container image registry API as a source of the image which allows the developers to push and pull their public container images.
+
+{op-system-base} Quay Container Registry::
```

Comment:
s/{op-system-base}/Redhat

Reply:
We must use {op-system-base} for the term Red Hat in the docs

Reply:
But in https://52812--docspreview.netlify.app/openshift-enterprise/latest/registry/index.html#openshift-registry-common-terms_registry-overview,  it's `RHEL Quay Container Registry`, it's not correct

Reply:
oh I see. I think you are right. I changed it to `Red Hat`